crates/forge_analyzer/src/definitions.rs

```diff
@@ -134,7 +134,7 @@ struct ModuleDefs {
 pub fn run_resolver(
     modules: &TiSlice<ModId, Module>,
     file_resolver: &ForgeResolver,
-    secret_packages: Vec<PackageData>,
+    secret_packages: &Vec<PackageData>,
```

Comment:
```suggestion
    secret_packages: &[PackageData],
```


Might as well pass in a slice if you're borrowing. You can fix the lifetime errors that you're getting by calling [slice::to_vec](https://doc.rust-lang.org/std/primitive.slice.html#method.to_vec) to convert it into a `Vec`. We can eliminate the clones of the `Vec` by refactoring the structs to take a slice instead of a `Vec`, but you don't have to do it in this PR if you don't want to.

Reply:
fixed. Thank you :)